src/first.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module utils/first
+ */
+
+/**
+ * Returns first item of an `iterable`.
+ *
+ * @param {Iterable.<*>} iterable
+ * @returns {*}
+ */
+export default function first( iterable ) {
+	const iteratorItem = iterable.next();
+
+	if ( !iteratorItem.done ) {
```

Comment:
For only one condition more logical is "if something than do something". Here there're two return statements so probably your version is a little bit more elegant.